runtime/v2/go.mod

```diff
@@ -40,12 +40,13 @@ require (
 	github.com/cockroachdb/redact v1.1.5 // indirect
 	github.com/cockroachdb/tokenbucket v0.0.0-20230807174530-cc333fc44b06 // indirect
 	github.com/cosmos/cosmos-proto v1.0.0-beta.5 // indirect
-	github.com/cosmos/iavl v1.3.0 // indirect
+	github.com/cosmos/iavl v1.0.0-beta.1.0.20240813194616-eb5078efcf9e // indirect
```

Comment:
Looks sketchy, why is it downgraded?

Reply:
I'm not sure... it persists after go.mod replace directive to store/v2 .. will investigate